File: python_ToSXs/python/05-uocsoboiso/103.py
arrInt = [14, 12, 3, 21, 6]
number = 3

result = "Không tồn tại"

for elm in arrInt:
    if (elm % number == 0) and (elm % (number + 1) == 0):
        result = "Tồn tại"

print(result)

File: cpp_e9zMZ/cpp/05-uocso-boiso/103.cpp
#include <iostream>
using namespace std;

int main() {
    int arrInt[] = {14, 12, 3, 21, 30};
    int number = 3;
    string result = "Không tồn tại";
    bool flagCheckExist = false;

    for (int elm : arrInt) {
        if (elm % number == 0 && elm % (number + 1) == 0) {
            flagCheckExist = true;
            break;
        }
    }

    if (flagCheckExist == true) {
        result = "Tồn tại";
    }

    cout << result;
    return 0;
}